.vib/gitea/goss/goss.yaml

```diff
@@ -0,0 +1,24 @@
+command:
```

Comment:
Using the [addr](https://github.com/goss-org/goss/blob/master/docs/manual.md#addr) Goss' test, I think we should be able to check the `ssh` endpoints both in the svc port and container port.